daemon/graphdriver/driver_unsupported.go

```diff
@@ -1,7 +1,14 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
@jhowardmsft why not keep windows unsupported? Would that prevent compiling? This PR doesn't bring any windows support for graphdriver, so I would prefer keeping it unsupported on windows while refactoring linux-specific things to make it easier for the future PR :)